homework5/task2.py

```diff
@@ -0,0 +1,20 @@
+import pickle
+
+with open('test/data/tuples.txt', 'r+b') as file:
+    data_tuple = pickle.load(file)
+count = 0
+for item in data_tuple:
+    count += 1
```

Comment:
why do you need this?

Reply:
@bohdandovbsyh bcs i forgot about enumerate and index, fixed now